doc/versions.md

```diff
@@ -2,21 +2,38 @@
 
 Hazelcast .NET brings the full power of the [Hazelcast](https://hazelcast.com) high-performance, in-memory computing platform to the Microsoft .NET ecosystem. The [Hazelcast .NET Client](https://hazelcast.com/clients/dotnet/) allows you to elastically scale your .NET caches at high read speeds, to access all of Hazelcast data structures such as distributed maps, queues, topics and more. All, with enterprise level security through SSL and mutual authentication.
 
-This page indexes documentation for all versions.
+## .NET support
 
-## Current version
+The following table defines the supported .NET versions for the various Hazelcast .NET Client versions.  
+Note that .NET Framework runs on Windows exclusively, whereas .NET Core and .NET 5.0+ run on Windows, Linux and MacOS.
+
+|Version|.NET Framework 4.5-4.6.1|.NET Framework 4.6.2-4.8|.NET Core 2.1|.NET Core 3.1 (LTS)|.NET 5.0|.NET 6.0 (LTS)|
```

Comment:
I know Microsoft won't support 4.6.1 soon but .NET Standard 2.0 covers 4.6.1, and Hazelcast client works on .NET standard 2.0, so 4.6.1, too. Isn't it a bit confusing? Should we mention on that?